database/migrations/2018_11_16_200157_add_secondary_causes_run_id_impact_doc_and_make_causes_nullable_to_campaigns_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddSecondaryCausesRunIdImpactDocAndMakeCausesNullableToCampaignsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('campaigns', function (Blueprint $table) {
+            $table->string('secondary_causes')->index()->after('cause')->nullable()->comment('Secondary causes associated with legacy campaigns');
+            $table->integer('campaign_run_id')->index()->after('secondary_causes')->nullable()->comment('Campaign Run Id to reference when we run script to update signup and post ids after legacy campaign migration. This column can be deleted once migration and update script are complete.');
+            $table->string('impact_doc')->index()->after('campaign_run_id')->nullable()->comment('URL to proof of impact doc.');
```

Comment:
We don't need an index on the `impact_doc` column since it won't be queried against.